documentation/modules/creating-migration-plan-2-6-3.adoc

```diff
@@ -55,6 +55,8 @@ To specify a different root device, in the *Settings* section, click the Edit ic
 {project-short} uses the following format for disk location: `/dev/sd<disk_identifier><disk_partition>`. For example, if the second disk is the root device and the operating system is on the disk's second partition, the format would be: `/dev/sdb2`. After you enter the boot device, click *Save*.
 +
 If the conversion fails because the boot device provided is incorrect, it is possible to get the correct information by looking at the conversion pod logs.
++
+include::snip_vmware_esxi_nfc.adoc[]
```

Comment:
I think this one should drop since we don't mention setting the ESXi network here (right?)

Reply:
Removed.

Reply:
Removed.